lib/backup/composer.rb

```diff
@@ -0,0 +1,66 @@
+require 'open3'
+
+module Backup
+  class Composer
```

Comment:
@ikappas Would it be possible to subclass `Backup::Files`, and override all its methods to do the default think for `public/p/`, and _also_ backup `public/packages.json`? I agree that `Backup::Files` doesn't (currently) need to support an array of targets.